### DISCUSSION TITLE
Lance

### DISCUSSION THREAD

**Discussion by tekumara** [tonbo-io/tonbo#161] [Ideas]
Hiya! I wondered if you'd considered lance as an alternative file format to parquet? It's optimised for random access.

https://github.com/lancedb/lance

**Comment by KKould** [comment]
Hi! @tekumara

sorry for the late reply. We didn't pay attention to the content in discussions. This is a very interesting idea. Most of tonbo's current designs serve OLAP for large-scale data scanning. At the same time, parquet is more mainstream in the arrow ecosystem. Of course, it is also meaningful to support other types of files such as lance. Maybe this will be our future features.
